Northstar.CustomServers/mod/scripts/vscripts/sh_progression.nut

```diff
@@ -88,21 +88,26 @@ bool function ClientCommand_SetProgression( entity player, array<string> args )
 
 /// Resets a specific Titan's Aegis rank back to `0`
 /// * `player` - The player entity to perform the action on
-/// * `args` - The arguments passed from the client command. `args[0]` should be an integer corresponding to the index of the Titan to reset.
+/// * `args` - The arguments passed from the client command. `args[0]` should be a string corresponding to the chassis name of the Titan to reset.
+/// * `chassis name` - Valid chassis are: Ion, Tone, Vanguard, Northstar, Ronin, Legion and Scorch.
 ///
 /// Returns `true` on success and `false` on missing args.
 bool function ClientCommand_ResetTitanAegis( entity player, array<string> args )
 {
 	if ( !args.len() )
 		return false
 	
-	int suitIndex = args[0].tointeger()
+	string titanRef = args[0]
```

Comment:
See above comment, might need to convert this to lowercase

Reply:
`titanRef` is a variable name i adopted from Respawn's code, they use it almost everywhere when it comes to referring titans in pdata stuff.

Reply:
Got what you mean, almost forgot `.tolower()` is a thing in squirrel.